src/dmd/ob.d

```diff
@@ -1676,7 +1676,7 @@ void genKill(ref ObState obstate, ObNode* ob)
 
             override void visit(NewExp e)
             {
-                Type tb = e.newtype.toBasetype();
+                // Type tb = e.newtype.toBasetype();
```

Comment:
No need to comment things out, just remove them.

Reply:
Done